test/passing/tests/let_mutable.ml

```diff
@@ -0,0 +1,4 @@
+let _ =
```

Comment:
It would be good to have a few more test cases (for example, how it works with `let mutable ... and ...`).